### PR TITLE
*: use std/slices to replace exp/slices

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: ref #45933

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: testkit/result.go [hunk 1]
```diff
@@ -86,15 +86,15 @@ func Rows(args ...string) [][]interface{} {
...
 			if a[i] < b[i] {
-				return true
+				return -1
 			} else if a[i] > b[i] {
-				return false
+				return 1
 			}
 		}
-		return false
+		return 1
```
Comment (lance6716) [inline testkit/result.go:97]: should we return 0?

Comment (hawkingrei) [reply testkit/result.go:97]: we need to add more logic like checking the length of slice.

Comment (hawkingrei) [reply testkit/result.go]: I have refactored it.

**Review by Defined2014 — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]

**Pull request merged**